sched/db_dump.cpp

```diff
@@ -765,12 +796,38 @@ int ENUMERATION::make_it_happen(char* output_dir) {
             exit(retval);
         }
         break;
+    case TABLE_USER_DELETED:
+        n = 0;
+        while (1) {
+            retval = user_deleted.enumerate("order by userid", true);
```

Comment:
Why do you use `mysql_use_result()` here (via `enumerate(clause, true)`) but `mysql_store_result()` [below](a1147ebff483de86a4970ff9492cbb91R854)?

Reply:
and it is now changed

Reply:
As long as it's an informed decision I'm fine with either.